app/__init__.py

```diff
@@ -2,17 +2,18 @@
 from flask import Flask
 from .api.V1 import version_1
 from .api.V2 import version_2
-from app.db_config import create_orders_table,create_users_table
+from app.db_config import create_orders_table,create_users_table, create_super_admin
 import os
 from flask_jwt_extended import JWTManager
 
 
-def create_app():
+def create_app(config):
     """Crete app fuction."""
     app = Flask(__name__)
     app.secret_key = os.getenv("secret_key") or "brian"
     create_orders_table()
     create_users_table()
+    create_super_admin()
```

Comment:
these `create_orders_table(), create_users_table(), create_users_table(), create_super_admin()` should not be done during create app. This means tables are created everytime you start the application.